src/sentry/hybridcloud/models/apikeyreplica.py

```diff
@@ -30,3 +31,20 @@ class Meta:
         db_table = "hybridcloud_apikeyreplica"
 
     __repr__ = sane_repr("organization_id", "key")
+
+    @property
+    def is_active(self):
+        return self.status == ApiKeyStatus.ACTIVE
+
+    def get_allowed_origins(self):
+        if not self.allowed_origins:
+            return []
+        return list(filter(bool, self.allowed_origins.split("\n")))
+
+    def get_audit_log_data(self):
```

Comment:
Do we have any automation to ensure that replica models like this have the same interface as the source models?

Reply:
There is some decent coverage in existing API tests, but specifically wrt `get_allowed_origins` I added our decorator https://github.com/getsentry/sentry/pull/56976/commits/4b8ac7ff1e92bbf7e524ac32e3fc2626ab2a84b9 so that it is covered for the region case as well.